docs/how-to-deploy-scalardb.md

```diff
@@ -1,8 +1,16 @@
-# How to deploy ScalarDB Server
+# [Deprecated] How to deploy ScalarDB Server
+
+{% capture notice--info %}
+**Note**
+
+ScalarDB Server is now deprecated. Please use [ScalarDB Cluster](./how-to-deploy-scalardb-cluster.md) instead of it.
```

Comment:
Just a minor suggestion here. (Changes made here are consistent across other docs in this PR.)

```suggestion
ScalarDB Server is now deprecated. Please use [ScalarDB Cluster](./how-to-deploy-scalardb-cluster.md) instead.
```